src/stub_renderer.cc

```diff
@@ -0,0 +1,88 @@
+/*************************************************************************
+ * Copyright (c) 2016 Fran�ois Trudel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+*************************************************************************/
+#include "stub_renderer.hh"
```

Comment:
This should be renamed to something else. This became something definitely different than a stub.